### DISCUSSION TITLE
Bulk edit color for all bookmarks from a list

### DISCUSSION THREAD

**Discussion by druxstr** [organicmaps/organicmaps#5301] [Ideas]
You may want all bookmarks from a list to have the same color. 

Right now, you can only do so by editing each bookmark manually.

Use case : 

> You display 2 large lists of bookmarks (+100), but every bookmark has the same color and icon. You want to give all bookmarks from your 1st list a distinct color.

This would be especially useful when importing large lists of bookmarks, as you can't set the default color before import.

**Comment by biodranik** [comment]
There should be already related issues that you may update.
